musicproject/settings.py

```diff
@@ -23,9 +23,9 @@
 SECRET_KEY = 'django-insecure-ieid&zz^!10-@rw1vdo!@*wnccb367dw@v@ema%i8_)b9%6fvj'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
Discard this file from commit. no need on server.

Reply:
I should not commit settings.py, or DEBUG = False?
As I am just begging, I really need a little explanation here on what I should not commit here? could you please tell me a little more on this.

Reply:
Hi, So what ever the changes you have made in this file is not required. you can remove this file changes from your commit. settings files can be added if you are modifying anything on server like new app or config, but not local server changes like adding local host to allowed host or changing settings which are not necessary for server.

Reply:
distinct in search functionality

# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
#
# Date:      Sun Aug 27 22:18:57 2023 +0530
#
# On branch search_music
# No commands done.
# Next commands to do (7 remaining commands):
#    pick d4bc88a Update settings.py
#    pick 6ea2d25 Update settings.py
# You are currently editing a commit while rebasing branch 'search_music' on '2578a98'.
#
# Changes to be committed:
#       modified:   music/views.py
#
# Changes not staged for commit:
#       modified:   music/templates/music/index.html
#       modified:   music/templates/music/search_result.html


I am here and I don't know how to exit this page. I was removing the commit I made in settings.py and stuck here. I have watched a few merge conflit videos still getting same problem. if you  have any suggestion, I would love to hear

Reply:
copy paste setting file from repo code in your setting file and push it.